### PR TITLE
Selenium: not check known issues in selenium page objects

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR moves checking known issues from selenium page objects to tests.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/11688

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1092/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1092//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11742
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Wizard.java [hunk 1]
```diff
@@ -360,13 +358,8 @@ public void waitOpenProjectConfigForm() {
...
 
   public void waitCloseProjectConfigForm() {
-    try {
-      new WebDriverWait(seleniumWebDriver, LOADER_TIMEOUT_SEC)
-          .until(ExpectedConditions.invisibilityOfElementLocated(By.id(Locators.MAIN_FORM_ID)));
-    } catch (TimeoutException ex) {
-      // remove try-catch block after issue has been resolved
-      fail("Known issue https://github.com/eclipse/che/issues/10713", ex);
-    }
+    new WebDriverWait(seleniumWebDriver, LOADER_TIMEOUT_SEC)
```
Comment (Ohrimenko1988) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Wizard.java]: Please, use "SeleniumWebDriverHelper#waitInvisibility" method:
```
seleniumWebDriverHelper.waitInvisibility(By.id(MAIN_FORM_ID), LOADER_TIMEOUT_SEC);
```

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NavigationBar.java [hunk 2]
```diff
@@ -74,13 +72,7 @@ public NavigationBar(SeleniumWebDriver seleniumWebDriver) {
...
 
   public void waitNavigationBar() {
-    try {
-      new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-          .until(visibilityOf(navigationBar));
-    } catch (TimeoutException ex) {
-      // remove try-catch block after issue has been resolved
-      fail("Known issue https://github.com/eclipse/che/issues/8490");
-    }
+    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC).until(visibilityOf(navigationBar));
```
Comment (Ohrimenko1988) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NavigationBar.java]: Please, use "SeleniumWebDriverHelper#waitVisibility" method:
```
seleniumWebDriverHelper.waitVisibility(navigationBar, LOAD_PAGE_TIMEOUT_SEC);
```

**Comment by SkorikSergey** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1095/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1095//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11742
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by SkorikSergey** [comment]
сi-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1103/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1103//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11742
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Review by dmytro-ndp — APPROVED** [review]

**Review by Ohrimenko1988 — APPROVED** [review]

**Pull request merged**